src/FormattingSettingsService.ts

```diff
@@ -50,31 +56,27 @@ export class FormattingSettingsModel {
                 uid: objectName + "-group"
             }
 
-            let formattingCard: visuals.FormattingCard = {
-                displayName: card.displayName,
-                groups: [formattingGroup],
-                uid: objectName,
-                analyticsPane: card.analyticsPane
-            }
+            let formattingCard: visuals.FormattingCard = card.getFormattingCard(objectName, formattingGroup, this.localizationManager);
 
             formattingModel.cards.push(formattingCard);
+
+            // In case formatting model contains multiple categories selectors which they use same capabilities object name and property name, 
```

Comment:
typo ? ( "which they use same..." )

Reply:
I fixed and added more explanation to the comment